### ISSUE TITLE
Uncaught TypeError: Cannot read properties of null (reading 'innerText')

### ISSUE THREAD

**Issue by Cartoonzzz** [nmanumr/quizsliver#26]
ok so when portal is opened with the latest version, if the extension is on, the course description cannot be opened. 
if it is turned off and then the course description is opened after refreshing the page, it will open.
and the extension is NOT WORKING PROPERLY 
Most of the answers are not found in database. You,ll have to do them on your own.
some which are found are wrong.

![Screenshot (947)](https://user-images.githubusercontent.com/34788500/196161784-349d3d85-5f48-4b01-8b0f-239ac0b76139.png)

**Comment by nmanumr** [comment]
This `TypeError` thing is something I can fix but not sure about MCQs not found or wrong options selected. For that I need MCQs question itself as examples to verify. I have some theories here, I'm not gonna list all of them here as they can be used to crack this extension itself, but all of them require some major refactoring of quiz sliver, I discussed some of that improvement here: https://github.com/nmanumr/quizsliver/discussions/13

The only hurdle in improving the quiz sliver is that I don't have any motivation to do this significant refactoring. Initially, the model for quiz sliver I had in mind was that we would ultimately have a community which will continue to maintain this project which never happened and I had to maintain the whole project all alone.

Then, I thought of sponsorware which means that the users have to pay in some way to use the extension which seems much more feasible on the surface but I don't have enough data to make a decision on that.

Long story short, the project is in maintenance mode for now and I have to make a decision whether to stop maintaining this project or to invest in refactoring the project.
